grpc/server/src/CMakeLists.txt

```diff
@@ -70,12 +36,16 @@ target_include_directories(
     ${PLUGINS_DIR}
 )
 
-target_link_libraries(
-    dronecore_server
+add_executable(backend_bin
```

Comment:
How about renaming it as `backend_server` ?

Reply:
But then should the library be called `libdronecore_backend_server`? I assumed that "backend" was enough, as a "grpc backend" is necessarily a server somehow :D.

What do you think?

Reply:
If "backend" is the name used in practice, I am fine with it.